### DISCUSSION TITLE
prerequsite for pyaedt in Linux

### DISCUSSION THREAD

**Discussion by IkeChoice** [ansys/pyaedt#3698] [Q&A]
I've got an error  when I try to open an EDB

Traceback (most recent call last):
  File "../../../../../pyaedt/generic/clr_module.py", line 22, in <module>
    import dotnetcore2
ModuleNotFoundError: No module named 'dotnetcore2'

I guess I need "dotnetcore2." Am I correct?

What other python module should be installed in my Linux (Centos7) machine?

Thanks

**Comment by Samuelopez-ansys** [comment]
@svandenb-dev

**Comment by Samuelopez-ansys** [comment]
@SMoraisAnsys

**Comment by SMoraisAnsys** [comment]
@IkeChoice Hi, thanks for your interest in `pyaedt`.
Can you tell me how you installed `pyaedt` ? As a workaround for your problem, you could try to use the python interpreter embedded in AEDT.

**Comment by Samuelopez-ansys** [comment]
@IkeChoice Any update?
